### DISCUSSION TITLE
You got verified?!

### DISCUSSION THREAD

**Discussion by TechStudent10** [KendallDoesCoding/KendallDoesCoding#4] [Q&A] [answered]
**HOL UP**

![image](https://user-images.githubusercontent.com/76978184/151006826-0e373871-fad9-47ac-ba1b-36356277772a.png)

How!?

**Comment by KendallDoesCoding** [comment, marked as answer]
> **HOL UP**
> 
> ![image](https://user-images.githubusercontent.com/76978184/151006826-0e373871-fad9-47ac-ba1b-36356277772a.png)
> 
> How!?

LOL, it just means verified domain. Like, if I add youtube.com it will say Approved but not verified, it doesnt mean I'm verified from github, I dont think that even exists lol.

**Reply by TechStudent10** [reply]
So you verified your domain? Got it.

**Reply by KendallDoesCoding** [reply]
Yes, tap "Verified" it will give you details, I just needed to add a CNAME or TXT record, I forgot what it was exactly.

**Reply by KendallDoesCoding** [reply]
![image](https://user-images.githubusercontent.com/84712013/151027915-5c1e8ec0-cd8e-4a43-b329-13dc0636e997.png)

**Reply by TechStudent10** [reply]
Cool

**Reply by KendallDoesCoding** [reply]
Indeed, That's one of the main reasons I made this organization, but also so my site websites are organized properly lol

**Reply by TechStudent10** [reply]
haha
